pkgs/development/python-modules/qiskit-ibmq-provider/default.nix

```diff
@@ -0,0 +1,68 @@
+{ lib
+, pythonOlder
+, buildPythonPackage
+, fetchPypi
+, nest-asyncio
+, pythonWebsockets
+, qiskit-terra
+, requests
+, requests_ntlm
+  # check inputs
+, pytestCheckHook
+, vcrpy
+, pproxy
+}:
+
+let
+  websockets7 = (
+    pythonWebsockets.overrideAttrs (
+      oldAttrs: rec {
+        version = "7.0";
+
+        src = fetchPypi {
+          inherit (oldAttrs) pname;
+          inherit version;
+          sha256 = "08e3c3e0535befa4f0c4443824496c03ecc25062debbcf895874f8a0b4c97c9f";
+        };
+      }
+    )
+  );
+in
```

Comment:
not allowed in python-modules, will create incoherent imports if another package picks up the default version. If this package just needs to export the executable, then we can move this package out of `python-modules`. But overriding is not allowed in `python-modules`